### PR TITLE
julia/docs: more DRY on page rendering

### PR DESCRIPTION
**Pull request by iblis17**
address https://github.com/apache/incubator-mxnet/pull/16363#issuecomment-537904494

### CODE REVIEW

**Comment by iblis17** [comment]
well, I think the CI failure is unrelated.

**Comment by aaronmarkham** [comment]
> well, I think the CI failure is unrelated.

Reported it here: https://github.com/apache/incubator-mxnet/issues/16443

I'll restart the test too.

File: julia/docs/src/api/ndarray.md [hunk 1]
```diff
@@ -70,21 +70,5 @@ In the following example `y` can be a `Real` value or another `NDArray`
...
-  "ndarray/io.jl",
-  "ndarray/linalg.jl",
-  "ndarray/reduction.jl",
-  "ndarray/remap.jl",
-  "ndarray/show.jl",
-  "ndarray/statistic.jl",
-  "ndarray/trig.jl",
-  "ndarray/type.jl",
-]
+Pages = listpages("ndarray")
```
Comment (aaronmarkham) [inline julia/docs/src/api/ndarray.md]: So if anything gets added you don't have to list it out manually?

Comment (iblis17) [reply julia/docs/src/api/ndarray.md]: yeah, that function `list`s the dir `src/ndarray`.

**Pull request merged**